### ISSUE TITLE
Adding French Translation

### ISSUE THREAD

**Issue by aarlaud** [saving-satoshi/saving-satoshi#1221]
Hey team,

if you're game, I'll get some french translation going. I'll translate the copy from the english copy bit by bit. I'll push a branch and drop it in this issue if more people want to contribute.

cheers !

**Comment by benalleng** [comment]
Absolutely! we have a translation Readme here for contributing. https://github.com/saving-satoshi/saving-satoshi/tree/dev/i18n#readme

**Comment by satsie** [comment]
Closing this since it's not actively being worked on but happy to reopen if anyone wants to pick this up.
